### DISCUSSION TITLE
Unable to Access Pageclip Benefits with GitHub Education

### DISCUSSION THREAD

**Discussion by JewelPaul** [community/community#142261] [GitHub Education]
### Hi everyone,

Hello,

I am a GitHub Education user, and I recently attempted to sign up for Pageclip, which I understand is included as part of the GitHub Education Pack. While I successfully signed up on Pageclip using my GitHub Education credentials, I am unable to access the plan or benefits associated with GitHub Education on the Pageclip platform.

Could you please assist me in resolving this issue or provide guidance on how to properly access the Pageclip benefits?

Thank you for your assistance.

Best regards,
Jewel Paul

**Comment by davevad93** [comment]
Hi @JewelPaul ,

In case of issues when redeeming the `Student Developer Pack` offers, it's recommended to contact support by opening a ticket [here](https://support.github.com/contact/education), select `I am a verified student having trouble redeeming my Global Campus offers` and explain your problem with Pageclip.

**Comment by JewelPaul** [comment]
cant you contact them ? i have to use pageclip but i cant, cant you helo me ? they are not replying what shall i do ? just tell me product key or anything so that it works. its a request.

**Reply by davevad93** [reply]
Hey @JewelPaul , sorry I don't work for GitHub, support can take a while to reply because they have a lot of daily requests to deal with, I understand it's frustrating but be patient, I hope they get back to you as soon as possible! 👍

**Comment by balintdecsi** [comment]
Hi, is there any update? I'm running into the same problem.

**Comment by JewelPaul** [comment]
No update yet. Still same issue

Jewel Gabriel Paul
✉️ ***@***.***
